app/javascript/components/modals/CommunityLaunchModal.tsx

```diff
@@ -52,84 +52,73 @@ export const FirstTimeModal = ({
       open={open}
       {...props}
       onClose={() => {}}
```

Comment:
```suggestion
      onClose={() => null}
```
https://github.com/typescript-eslint/typescript-eslint/blob/v4.33.0/packages/eslint-plugin/docs/rules/no-empty-function.md

Reply:
also some unused variables in this file. e.g. `Avatar`, line 7 (`import { User } from '../types'`)